<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicPermutationGroupWidget</class>
 <widget class="QGroupBox" name="BasicPermutationGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Permutation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="inputText">
     <property name="placeholderText">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="showKeyCheckBox">
     <property name="text">
      <string>Show key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="directionLabel">
     <property name="text">
      <string>Permutation direction: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="directionComboBox">
     <item>
      <property name="text">
       <string>Column permutation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Row permutation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grid permutation (both)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="encryptButton">
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="decryptButton">
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextEdit" name="outputText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="keyLine">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Key</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
